server/modules/worker.js

```diff
@@ -0,0 +1,14 @@
+import client from '../helpers/redis';
+
+client.set('startTime', JSON.stringify(new Date()));
```

Comment:
The piece of code here:
```
client.set('startTime', JSON.stringify(new Date()));
```
would execute as soon as this file(`worker.js`) is imported somewhere else, consequently, the `startTime` is not going to tell when the worker actually started working but when this file(`worker.js`) was imported/processed. 

A possible way to fix this is to make the `worker` expose a method to write the `startTime`, consequently, the `worker` can be used like this
```
...
worker.init() // ===> Writes the `startTime` and do other setup operations
worker.doStuff() // ===> Perform the actual automation operation
...
```
👆 is just and idea.

Reply:
make sense